bolt/lib/Core/BinaryFunction.cpp

```diff
@@ -389,6 +389,13 @@ bool BinaryFunction::isForwardCall(const MCSymbol *CalleeSymbol) const {
   }
 }
 
+void BinaryFunction::dump(const MCInst &Inst) const {
+  if (!BC.InstPrinter)
```

Comment:
It seems to me it is more of a BC interace, not BF. Also suggest you to write something on if instprinter is not ready.
Anyway it needs to be used somehow, it could not be just merged without proper usage, tests & etc

Reply:
Oh, thanks for your suggestion. I will try to modify it to BC interface and add test cases or usage, for testing purposes.